src/api/spec/models/project_spec.rb

```diff
@@ -469,4 +469,56 @@
       it { expect(Project.deleted?(project.name)).to be_truthy }
     end
   end
+
+  describe '#render_relationships' do
+    let!(:project) { create(:project) }
```

Comment:
`!` is not needed for project, group and role

Reply:
@Ana06 @ChrisBr I prefer to just include the `!` if I know that the block needs to be eagerly evaluated even though it might not be necessary. Otherwise it means that you always need to check the code to see if `project` gets called at some point which just seems like extra time and effort spent without any real gain?

Reply:
Ok, I'm fine with that! Go ahead :+1: